src/libs/dutil/inc/strutil.h

```diff
@@ -282,7 +282,7 @@ HRESULT DAPI StrAllocStringToLowerInvariant(
 
 HRESULT DAPI StrArrayAllocString(
     __deref_inout_ecount_opt(*pcStrArray) LPWSTR **prgsczStrArray,
-    __inout LPUINT pcStrArray,
+    __inout UINT* pcStrArray,
```

Comment:
New style?

Reply:
`LEAN_AND_MEAN` does not define `LPUNIT`. Surprised me.